### DISCUSSION TITLE
A component like the DALL·E 3 response in chatgpt.

### DISCUSSION THREAD

**Discussion by aisensiy** [saadeghi/daisyui#2676] [Q&A] [answered]
https://github.com/saadeghi/daisyui/assets/661860/e5f698ad-7478-4627-a378-d17ee3a8fd4b

I want to generate a component like the chatgpt dalle 3 using daisyui, any idea about this?

**Comment by saadeghi** [comment, marked as answer]
Assuming you're talking about the image modal,
you can use a daisyUI modal, then you can put a daisyUI drawer inside the modal so it can have a collapsible sidebar.
